CMakeLists.txt

```diff
@@ -748,6 +748,10 @@ if (HDF5_BUILD_SUBFILING)
   if (NOT HDF5_ENABLE_PARALLEL)
     message (FATAL_ERROR "Subfiling VFD requires a parallel HDF5 build")
   endif()
+  CHECK_INCLUDE_FILE("stdatomic.h" HAVE_STDATOMIC_H)
```

Comment:
CMake will need this added to the h5pubconf.h.in file

Reply:
You'll probably want to add that to this PR?

Reply:
One needs to add standard C headers? What is the purpose of h5pubconf.h.in?